### PR TITLE
1.97 release notes v0 draft STO-7649

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
### Preview links
- [Version 1.97](https://666702c7a97c2f49808133c8--harness-developer.netlify.app/release-notes/security-testing-orchestration#version-197)
- Semgrep scanner reference > [Important notes for running Semgrep scans in STO](https://666702c7a97c2f49808133c8--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/semgrep/semgrep-scanner-reference#important-notes-for-running-semgrep-scans-in-sto) (second bullet point)

### Merge requirements
PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: release-notes/security-testing-orchestration.md [hunk 1]
```diff
@@ -22,6 +22,58 @@ These release notes describe recent changes to Harness Security Testing Orchestr
 
 :::
 
+## June 2024
+
+### Version 1.97
+
+#### New feature
+
+- The STO API is now generally available and publicly documented. For more information, go to [STO](https://apidocs.harness.io/tag/Exemptions) in the Harness API documentation. (STO-5281)
```
Comment (aklaft) [inline release-notes/security-testing-orchestration.md:31]: Maybe we want to mention it's the exemption api?

**Review by aklaft — APPROVED** [review]

File: release-notes/security-testing-orchestration.md (hunk 1 above)
Comment (douglas-j-bothwell) [reply release-notes/security-testing-orchestration.md:31]: Unfortunately there's no way to link to the STO heading so I linked the first subheading.

File: release-notes/security-testing-orchestration.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Security Testing Orchestration release notes
 sidebar_label: Security Testing Orchestration
 description: Provides an overview of new features and fixed issues.
-date: 2024-05-23T10:00
+date: 2024-06-05T10:00
```
Comment (brian-f-harness) [inline release-notes/security-testing-orchestration.md:5]: Was this released last week, or should this be updated?

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Pull request merged**